src/main/java/seedu/taskify/ui/TaskCard.java

```diff
@@ -52,7 +52,19 @@ public TaskCard(Task task, int displayedIndex) {
         id.setText(displayedIndex + ". ");
         name.setText(task.getName().fullName);
         description.setText(task.getDescription().value);
-        status.setText(task.getStatus().toString());
+        String statString = task.getStatus().toString();
+        status.setText(statString);
+        System.out.println(statString);
```

Comment:
Just to confirm, is System.out.println() supposed to be there?

Reply:
Nope that was for debugging, good catch